binder/environment.yml

```diff
@@ -22,3 +22,28 @@ dependencies:
     - pycountry-convert==0.7.2
     - openpyxl==3.0.9
     - networkx==2.6.3
+    - cycler==0.10.0            # via matplotlib
+    - kiwisolver==1.2.0         # via matplotlib
+    - matplotlib==3.2.1
+    - numpy==1.18.5
+    - pandas==1.0.4
+    - pyparsing==2.4.7          # via matplotlib
+    - python-dateutil==2.8.1    # via matplotlib, pandas
+    - pytz==2020.1              # via pandas
+    - scipy==1.4.1              # via seaborn
+    - seaborn==0.10.1
+    - six==1.15.0               # via cycler, python-dateutil
+    - sklearn==0.0
+    - imblearn==0.0
+    - colorama==0.4.4
+    
+   - sphinx:
+        config:
+          latex_elements:
+          # Note this line cheats the table parameter into the documentclass[] parameter
+          papersize: letterpaper,table
+          # These lines add the desired packages to the latex build
+          preamble: |+
+            \usepackage{xcolor}
+            \usepackage{booktabs}
+            \usepackage{multirow}
```

Comment:
I'll remove the Sphinx and the rest.  The Sphinx was just an experiment; stand by while I make that change now.

Reply:
Done.  I've edited the environment.yml to shorten it.  See whether that meets your requirement and let me know.

Reply:
Hi Luke, After the workday, I got a chance to review the environment file. I can see now you had four packages below the PIP line that I had taken out. I restored those to the file and retained the colorama and imblearn as new additions. I /*think/* that should be in keeping with your intentions for the file now but if not, let me know. I have repeated this message in your Slack TDS inbox as well  Thanks.

Reply:
Oh, and in case it was not obvious, I did a minor change over the weekend to the axis labels of the scatter plots and sized the bar charts a little bit better to fit in all of the words. If you were taking copies of the story, you may want to head back to the original document to get the best version.  I've read the entire story again in full today and can foresee no future changes on my side.  Thanks Luke.